src/main/java/com/amazon/reviews/service/FileParsingService.java

```diff
@@ -0,0 +1,8 @@
+package com.amazon.reviews.service;
+
+import com.amazon.reviews.model.dto.ReviewDto;
+import java.util.List;
+
+public interface FileParsingService {
+    List<ReviewDto> parse(String path);
```

Comment:
Since it is an interface - you can parametrize it. You can have different dto to parse, so you can have different return List type

Reply:
Also, I think we need to have separate interfaces for reading and parsing

Reply:
I agree with Andrew on both his comments
you will call FileReaderService and then ParsingService from your another place of your program ([ApplicationListener](https://reflectoring.io/spring-boot-execute-on-startup/#applicationlistener), for example)